Makefile

```diff
@@ -56,8 +35,8 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-test: fmt vet envtest
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
```

Comment:
Why remove envtest and not use the `USE_EXISTING_CLUSTER` flag?

Reply:
I can't any pro in using envtest in this setting. Because anyway this flag must be set to true in order to run integration tests successfully. Maybe it's better to split unit and integration test directly in the `Makefile` so users can run the tests they wish. Does it make sense to you?

Reply:
Reverted integration with `kind` changes. Now only units test are included